templates/vars/aks/cftptl-intsvc.json

```diff
@@ -3,22 +3,22 @@
   "contentVersion": "1.0.0.0",
   "parameters": {
     "kubernetesVersion": {
-      "value": "1.17.9"
+      "value": "1.18.8"
     },
     "nodeMinCount": {
       "value": 8
     },
     "nodeMaxCount": {
-      "value": ""
+      "value": "10"
     },
     "enableAutoScaling": {
-      "value": ""
+      "value": "true"
     },
     "type": {
-      "value": "AvailabilitySet"
+      "value": "VirtualMachineScaleSets"
     },
     "vmSize": {
-      "value": "Standard_D8s_v3"
```

Comment:
why re we changing vmSize?

Reply:
We should not change the vmSize in this PR. Autoscaler is not dependent on the vmSzie

Reply:
reverted it